tests/unit/test_formatter.py

```diff
@@ -117,3 +117,15 @@ def test_query_parameters_list(self):
             """
         ).strip()
         assert res == expected
+
+    def test_query_presto_style_escapes_for_inserts(self):
```

Comment:
I copied this test and run it against the main branch and it passed. See my comment https://github.com/dbt-athena/dbt-athena/pull/71/files#r1033966525